ledger-service/http-json/src/itlib/scala/http/AbstractHttpServiceIntegrationTest.scala

```diff
@@ -1049,14 +1098,15 @@ abstract class AbstractHttpServiceIntegrationTest
 
   "exercise IOU_Transfer with unknown contractId should return proper error" in withHttpService {
     (uri, encoder, _, _) =>
-      val contractId = lar.ContractId("#NonExistentContractId")
+      val contractIdString = List.fill(66)("0").mkString("")
```

Comment:
```suggestion
      val contractIdString = "0" * 66
```
But I'm not sure of what is the purpose of this change, the existing name seems quite telling.

Reply:
I changed it as the test started failing complaining about `V0 contract ids being forbidden` on migrating away from ` sandbox-classic`

```scala

"CommandService Error, io.grpc.StatusRuntimeException: INVALID_ARGUMENT: Invalid argument: Command interpretation error in LF-DAMLe: Illegal Contract ID "#NonExistentContractId", V0 Contract IDs are forbidden. Details: N/A." did not include substring "Contract could not be found with id ContractId(#NonExistentContractId)"

```